src/main/java/org/traccar/web/client/utils/RoutePolylineFinder.java

```diff
@@ -73,15 +63,15 @@ public void onError(Request request, Throwable exception) {
     }
     
     public interface Callback {
-        void onResult(LonLat[] points, double[] distances);
+        void onResult(String points, double[] distances);
     }
     
     static class Result extends JavaScriptObject {
         protected Result() {}
         
-        public final native Object[] geometry(int i) /*-{
+        public final native String geometry(int i) /*-{
```

Comment:
Ponownie zakomentowany kod.
Nie mówię, żeby go usuwać, ale dlaczego?

Reply:
To nie jest komentarz.